src/vmm/src/lib.rs

```diff
@@ -301,6 +311,34 @@ impl VMM {
         Ok(())
     }
 
+    fn add_block_device(&mut self, cfg: &BlockConfig) -> Result<()> {
```

Comment:
Is it too early to add other parameters to this (for example the `MmioConfig`) or to split it in 2 fns and make it, this way, work for any device?

Reply:
The current plan is to convert several parts of the current functionality to be re-usable for all virtio devices in the ref-vmm in the upcoming net PR.